From e6625cbb822231232fa98e3d720af52c34224e17 Mon Sep 17 00:00:00 2001
From: Tejas Patel <tejas.patel1@einfochips.com>
Date: Wed, 19 Jan 2022 13:26:34 +0530
Subject: [PATCH 10/13] arch: arm64: configs: Increase CMA size

This patch increases CMA size, so that gstreamer can work on
higher resolution.

Change-Id: I84f71c60f78dd420f5f500ded46239c92fbd2a3c
Signed-off-by: Tejas Patel <tejas.patel1@einfochips.com>
Signed-off-by: Tanvi Chauhan <tanvi.chauhan@einfochips.com>
---
 arch/arm64/configs/imx_v8_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/imx_v8_defconfig b/arch/arm64/configs/imx_v8_defconfig
index ecfc7efac..f04d9d73c 100644
--- a/arch/arm64/configs/imx_v8_defconfig
+++ b/arch/arm64/configs/imx_v8_defconfig
@@ -964,7 +964,7 @@ CONFIG_CRYPTO_DEV_HISI_HPRE=m
 CONFIG_CRYPTO_DEV_AMLOGIC_GXL=m
 CONFIG_INDIRECT_PIO=y
 CONFIG_CRC_CCITT=m
-CONFIG_CMA_SIZE_MBYTES=32
+CONFIG_CMA_SIZE_MBYTES=256
 CONFIG_PRINTK_TIME=y
 CONFIG_DEBUG_INFO=y
 CONFIG_MAGIC_SYSRQ=y
-- 
2.17.1

